mllm/models/internlm2/tokenization_internlm2.hpp

```
@@ -0,0 +1,157 @@
+                     .setName("internlm2-tokenizer-i0")
+                     .alloc();
+
+    auto ptr = ret.ptr<int64_t>();
+    for (size_t i = 0; i < ids.size(); ++i) { ptr[i] = ids[i]; }
+
+    return ret;
+  }
+
+  ARGenerationOutputPast convertMessage(const InternLM2Message& prompt) {
+    auto applied_string = InternLM2Message::message_template;
+    size_t pos = applied_string.find("{{{prompt}}}");
```

Clang (14.0.6)

[error] variable 'pos' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)